bundle/Resources/public/css/modules/buttons.css

```diff
@@ -21,3 +21,48 @@
     background: #bbb;
     cursor: not-allowed;
 }
+
+.cof-btn.cof-btn--create {
+    text-transform: uppercase;
+    color: #fff;
+    font-size: .875rem;
+    font-weight: 700;
+    background: #3FB499;
+    padding: .7rem 2rem .7rem 2.6rem;
+    position: relative;
+}
+
+.cof-btn--create:before {
+    display: inline-block;
```

Comment:
It can be removed. Everytime you make something positioned absolutely it becomes an element with `display: block`.